composer.json

```diff
@@ -5,7 +5,7 @@
     "keywords": ["framework", "laravel"],
     "license": "MIT",
     "require": {
-        "php": "^7.3|^8.0",
+        "php": "^8.0",
```

Comment:
Os arquivos referentes ao Laravel Sail, estão de acordo para forçar essa versão ?

Reply:
Não me passou pela cabeça que tinha sail. @NicolasPereira pode mudar o setup do Sail para usar o PHP 8?

Reply:
cc: @NicolasPereira

Reply:
Vamos resolver no próximo PR do core.